Open Judge System/Web/OJS.Web/Controllers/SubmissionsController.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace OJS.Web.Controllers
+﻿using OJS.Data.Models;
```

Comment:
usings should be nested in namespace and separated by newline per source assembly, i.e
namespace ....
using System;
using System.Linq;
_newline_
using OJS.Data.Models;
using OJS.Data.Repositories.Base;
etc.